api/DocumentTimeline.json

```diff
@@ -27,17 +27,6 @@
                   "value_to_set": "true"
                 }
               ]
-            },
-            {
-              "version_added": "50",
-              "version_removed": "63",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.animations-api.core.enabled",
-                  "value_to_set": "true"
-                }
-              ]
             }
           ],
           "firefox_android": [
```

Comment:
Missing `version_removed` in this entry, but I can't comment on the appropriate line.